tests/userTest.js

```diff
@@ -592,6 +592,136 @@ describe('Main: Currently testing user management,', function () {
     })
   })
 
+  it('checking method : enableUser with an existing user', async function () {
+    const provider = createProvider()
+
+    await provider.uponReceiving('enable an existing user')
+      .withRequest({
+        method: 'PUT',
+        path: MatchersV3.regex(
+          '.*\\/ocs\\/v(1|2)\\.php\\/cloud\\/users\\/' + config.testUser + '\\/enable',
+          '/ocs/v1.php/cloud/users/' + config.testUser + '/enable'
+        )
+      }).willRespondWith({
+        status: 200,
+        headers: xmlResponseHeaders,
+        body: new XmlBuilder('1.0', '', 'ocs')
+          .build(ocs => {
+            ocs.appendElement('meta', '', (meta) => {
+              return ocsMeta(meta, 'ok', '100')
+            })
+          })
+      })
+
+    return provider.executeTest(async () => {
+      const oc = createOwncloud()
+
+      return oc.users.enableUser(config.testUser).then((status) => {
+        expect(status).toBe(true)
+      }).catch(error => {
+        expect(error).toBe(null)
+      })
+    })
+  })
+
+  it('checking method : disableUser with an existing user', async function () {
+    const provider = createProvider()
+
+    await provider.uponReceiving('disable an existing user')
+      .withRequest({
+        method: 'PUT',
+        path: MatchersV3.regex(
+          '.*\\/ocs\\/v(1|2)\\.php\\/cloud\\/users\\/' + config.testUser + '\\/disable',
```

Comment:
we figured out we don't need `v(1|2)` in the regex
```suggestion
          '.*\\/ocs\\/v1\\.php\\/cloud\\/users\\/' + config.testUser + '\\/disable',
```

Reply:
Just pushed that versioning update in the `userTests.js` file